cmake/ycm-0.3.1/find-modules/FindZFP.cmake

```diff
@@ -29,25 +29,27 @@ include(SelectLibraryConfigurations)
 
 find_path(ZFP_INCLUDE_DIR
           NAMES zfp.h
-          PATHS $ENV{ZFP_ROOT}/inc
+          PATHS $ENV{ZFP_DIR}/include
```

Comment:
No it's not, ROOT is the correct one, I don't see a `ZFPConfig.cmake` file in `ZFP_DIR`